proto/xudrpc.proto

```diff
@@ -400,7 +400,10 @@ message CloseChannelRequest {
   uint64 amount = 5 [json_name = "amount"];
 }
 
-message CloseChannelResponse {}
+message CloseChannelResponse {
+  // The id of the transaction.
```

Comment:
This should be plural, since it's returning multiple txids. And maybe it should say that it's one transaction per channel close.

Reply:
Changed as `The id of the transaction per channel close.`